notebook_examples/streaming-service-introduction.ipynb

```diff
@@ -47,6 +47,7 @@
     "This notebook uses the `kafka-python` client to connect to the Streaming service by leveraging the Streaming kafka-compatible API. Messages are consumed and produced using the `kafka-python` client.\n",
     "\n",
     "Developed on [Data Exploration and Manipulation](https://docs.oracle.com/en-us/iaas/data-science/using/conda-dem-fam.htm) for CPU on Python 3.7 (version 3.0)\n",
```

Comment:
just to check, this really is still a 3.7 notebook right?

Reply:
Yes